solutions/usaco-833.mdx

```diff
@@ -2,13 +2,105 @@
 id: usaco-833
 source: USACO Bronze 2018 Open
 title: Family Tree
-author: Danh Ta Chi Thanh
+author: Danh Ta Chi Thanh, Ryan Chou
 ---
 
 ## Solution
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_family_bronze_open18.html)
 
+By finding the closest common ancestor, and the distance from the ancestor and the two cows, we can uniquely identify their relationship.
+## Implementation
+**Time Complexity:** $\mathcal{O}(N^3)$
+
+
+<LanguageSection>
+<PySection>
+	
+```py
+import sys
+
+sys.stdin = open('family.in', 'r')
```

Comment:
can you change the quotes to be consistent (either all single quotes or all double quotes)

Reply:
sure